tools/install_ancm.ps1

```diff
@@ -82,9 +82,8 @@ function Invoke-UpdateAppHostConfig {
     ## Trick: if the worker process was running 32 bit modes, %windir%\System32 is actually redirected to %windir%\SysWOW64
     ## Hence we would always point to module to $inetSrvDirs.x64
     $aspNetCoreHandlerFilePath = Join-Path $inetSrvDirs.x64 "aspnetcorev2.dll"
-    Start-IISCommitDelay
-    $sm = Get-IISServerManager
 
+    $sm = Get-IISServerManager
```

Comment:
Why did you change install_ancm.ps1? As far as I know we don't use this one.

Reply:
We don't currently but vsts will be using it. I modified an existing script

Reply:
Considering this test for E2E, we should not use any custom script and use installer (msi) instead. In addition, using custom install script will cause only complexity. Please consider not to use this even for vsts. :-)

Reply:
No msi; windows hosting bundle sure.

Reply:
Talked to @jkotalik . Eventually we can use the windows hosting bundle msi to install both sharefx and ancm. Right now there's still work to be done to include ancmV2 in the hosting bundle. I will create a new work item for it.

Reply:
https://github.com/aspnet/AspNetCoreModule/issues/315